package/src/values/animation/timing/runTiming.ts

```diff
@@ -27,7 +27,11 @@ export const runTiming = (
   callback?: AnimationCallback
 ): SkiaAnimation => {
   const resolvedParameters = getResolvedParams(toOrParams, config);
-  const animation = createTiming(resolvedParameters, value, callback);
+  const resolvedCallback = (current: number) => {
```

Comment:
I appear to be able to reproduce the crash on iOS even on this branch, could that be? 🤔